### ISSUE TITLE
Provide a docker image

### ISSUE THREAD

**Issue by ToshY** [maaslalani/invoice#29]
**Is your feature request related to a problem? Please describe.**
I would like to see a docker image for this.

**Describe the solution you'd like**
Automatically publish a new docker image on each release.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by toshsan** [comment]
But why? A simple binary is not complicated enough?

**Comment by ToshY** [comment]
@toshsan I'm not sure if you're familiar with the concept of [containerized sofware](https://www.docker.com/resources/what-container/), but having a docker image would simplify its usage. One of the (many) pros is that it does not require the user to install go (and dependencies) on the host machine. You'd simply pull the (latest) image and run the container. 

The only "complicated" part would be to create a proper [`Dockerfile`](https://docs.docker.com/language/golang/build-images/#create-a-dockerfile-for-the-application) in order to build the image, and adding [publishing of the docker image](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images) upon release.

**Comment by toshsan** [comment]
Well dear containerized everything: this go software is also offered as standalone binary of 8MB under [release](https://github.com/maaslalani/invoice/releases). If you read beyond the marketing material from docker. There is no significant benefit of running a binary in and out of docker since there is no requirement of any other runtime libraries that could conflict with another system resource.

**Comment by ToshY** [comment]
Yes I can see the binaries, but that still does not mean a docker image is out of the question. This is just a feature request, and it's up to the owner to decide if it's interesting enough.
